__mocks__/generators/shared.ts

```diff
@@ -0,0 +1,102 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as imperative from "@zowe/imperative";
+import { ZoweDatasetNode } from "../../src/dataset/ZoweDatasetNode";
+import { ZoweUSSNode } from "../../src/uss/ZoweUSSNode";
+import * as path from "path";
+import * as globals from "../../src/globals";
+import * as vscode from "vscode";
+import { ValidProfileEnum } from "../../src/Profiles";
+
+export function generateImperativeSession() {
+    return new imperative.Session({
+        user: "fake",
+        password: "fake",
+        hostname: "fake",
+        protocol: "https",
+        type: "basic",
+    });
+}
+
+export function generateImperativeSessionWithoutCredentials() {
```

Comment:
In my PR I removed Imperative from all generator names...this function name was particularly long. Is it really necessary to specify that it is an imperative session?

Reply:
Yes, method name is long, but you understand which session exactly you create which, I think, is more important than short method names.

Reply:
can me maybe shorten it to `generateISessionNoCredentials` ?

Reply:
Sounds like a plan, I will add a fix for this name.